index.html

```diff
@@ -85,7 +85,12 @@
         var e = document.createElement('script');
         e.onerror = initError;
         e.src = '/static/webcomponents-lite.js';
-        document.head.appendChild(e);
+        if (document.readyState === 'loading' &&
```

Comment:
The document should always be in loading state as we are mid-parsing it. (unlike `webcomponent-loader.js` that could be loaded after the document is ready)